src/main/plugin-metadata/plugin-security.policy

```diff
@@ -1,32 +0,0 @@
-/*
```

Comment:
no needed with newer version of Java/Elasticsearch that don't have SecurityManager